app/models/work.rb

```diff
@@ -0,0 +1,22 @@
+class Work < ApplicationRecord
+    belongs_to :user
+    has_many :votes
```

Comment:
You can't delete a work that has votes! This is because you've created a link between the two tables at the database level, and deleting the work would leave behind invalid votes. You might want to check out the [dependent](http://guides.rubyonrails.org/association_basics.html#dependent) argument to `has_many`.